extensions/wikia/WikiFactory/SpecialWikiFactory_ajax.php

```diff
@@ -518,17 +518,15 @@ function axWFactoryClearCache()
 function axWFactorySaveVariable() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) The Cyclomatic Complexity of this function "axWFactorySaveVariable" is 22 which is greater than 20 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1541)